src/commands/revisionDraft/deployRevisionDraft/DeployRevisionDraftConfirmStep.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../../utils/localize";
+import type { IDeployRevisionDraftContext } from "./IDeployRevisionDraftContext";
+
+export class DeployRevisionDraftConfirmStep extends AzureWizardPromptStep<IDeployRevisionDraftContext> {
+    public async prompt(context: IDeployRevisionDraftContext): Promise<void> {
+        await context.ui.showWarningMessage(
+            localize('deployRevisionWarning', 'This will deploy a new revision to container app "{0}".', context.containerApp?.name),
```

Comment:
I think we should steer away from the word "revision" since we don't really expose that anywhere else. We should probably just have something to the effect that we're "Deploying to container app" or "Updating container app" or something.